tests/rule_based_profiler/data_assistant/test_onboarding_data_assistant.py

```diff
@@ -327,7 +327,7 @@ def test_onboarding_data_assistant_get_metrics_and_expectations_using_implicit_i
             "random_seed": 43792,
         },
         numeric_columns_high_precision_rule={
-            "round_decimals": 4,
+            "round_decimals": None,
```

Comment:
@NathanFarmer No, please let us leave it at 4 -- that was a good approach, meaningful for high precision.  Thank you.